### DISCUSSION TITLE
Is it possible to created a connected project with the API ?

### DISCUSSION THREAD

**Discussion by fletort** [testspace-com/support#33] [Q&A] [answered]
I did not success to create a connected projet (to a github project) with the webapi.
I try to define source_repo_url and also issues_provider. The first one is never take. (no present in a get response after)

I see that the site use another endpoint for that (a form one):  https://ORG.testspace.com/projects/connected

It is really possible to define connected projet with the webapi ?

**Comment by ivailop** [comment, marked as answer]
Yes, there is a way to create a connected project via the API. Notice it is not documented at this time, but is stable and safe to use...

Using the https://help.testspace.com/reference/web-api#create-a-project end point simple pass an extra `source_repo` argument formatted as:
```
source_repo: 'github:owner/repo'
``` 
where `owner/repo` is your GitHub repo's full-name.

**Reply by fletort** [reply]
Thanks !! this is working! I can add this in my script creating project.
